.github/workflows/nightly.yaml

```diff
@@ -6,12 +6,12 @@ on:
 jobs:
   build_and_test:
     name: Nightly build and test
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - name: deps
-        uses: ros-tooling/setup-ros@v0.4
+        uses: ros-tooling/setup-ros@v0.7
         with:
-          required-ros-distributions: humble
+          required-ros-distributions: rolling
```

Comment:
Change this (and below) to Jazzy?

Reply:
Done https://github.com/open-rmf/rmf/pull/479/commits/a27edac75abfc08021a175a383897bf50d54c14a